charts/kube-prometheus-stack/Chart.yaml

```
@@ -18,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 21.0.3
+version: 21.0.4
```

```suggestion
version: 21.0.5
```

Hey Vasily,

thanks for the effort you put into the chart until now :)
It's a pitty to see you leave as you one of the more active maintainers of the chart :(

Is there any way to convince you to stay?

Kind Regards
André

---

I've switched jobs almost 6 months ago and am no longer working in a related area. I haven't had adequate time in the last year to dedicate to this project and with me being out of the loop on related projects I cannot make good calls on what should or should not be in the charts.

---

To bad :( 
But good luck in the new job anyway :)